module-4/nmap/nmap.py

```diff
@@ -20,7 +20,15 @@
     # ... continue up to port 1000 as needed
 }
 
-def resolve_host_ip_from_name(host):
+def resolve_host_ip_from_name(host: str):
+    """resolve host ip
```

Comment:
better say something like "resolve ip address from given hostname" or similar, but feel free to change that docstring on your own behalf